packages/aws-cdk/bin/cdk.ts

```diff
@@ -105,10 +105,18 @@ async function parseCommandLineArguments() {
       .option('outputs-file', { type: 'string', alias: 'O', desc: 'Path to file where stack outputs will be written as JSON', requiresArg: true })
       .option('previous-parameters', { type: 'boolean', default: true, desc: 'Use previous values for existing parameters (you must specify all parameters on every deployment if this is disabled)' })
       .option('progress', { type: 'string', choices: [StackActivityProgress.BAR, StackActivityProgress.EVENTS], desc: 'Display mode for stack activity events' })
-      .option('rollback', { type: 'boolean', default: true, desc: 'Rollback stack to stable state on failure (iterate more rapidly with --no-rollback or -R)' })
+      .option('rollback', { type: 'boolean', default: undefined, desc: 'Rollback stack to stable state on failure (iterate more rapidly with --no-rollback or -R)' })
```

Comment:
I _think_ you can just remove the `default` part here completely, and it will do the same thing... can you try it, and let me know if it works?

Reply:
Can confirm that this works